libs/platform/src/lib/components/form/radio-group/radio-group.component.ts

```diff
@@ -205,52 +205,38 @@ export class RadioGroupComponent extends CollectionBaseInput implements AfterVie
      * subscribing to radio button clicked event
      */
     private _initContentRadioButtons(): void {
-        if (this.contentRadioButtons && this.contentRadioButtons.length > 0) {
-            let firstEnabledButtonIndex = -1;
-            this.keyboardEventsManager = new FocusKeyManager(this.contentRadioButtons)
-                .withWrap()
-                .withHorizontalOrientation('ltr');
-
-            this.contentRadioButtons.forEach((button, i) => {
-                this._setProperties(button);
-                this._selectUnselect(button);
-
-                // finding first enabled button to set tabIndex=0
-                if (!button.disabled && !this._disabled && firstEnabledButtonIndex < 0) {
-                    firstEnabledButtonIndex = i;
-                }
-                button.click.pipe(takeUntil(this.destroy$)).subscribe((ev) => this._selectedValueChanged(ev));
-            });
-            // accessibility requirement
-            if (!this._selected && this.contentRadioButtons && firstEnabledButtonIndex > -1) {
-                this.contentRadioButtons.toArray()[firstEnabledButtonIndex].setTabIndex(0);
-            }
-            this.onChange(this.value);
-        }
+        this._initialSetup(this.contentRadioButtons);
```

Comment:
Is it really necessary to have a method just to call another method? 
You could remove `_initContentRadioButtons` and move `this._initialSetup(this.contentRadioButtons);` to `ngAfterViewInit` hook

Reply:
fixed